### PR TITLE
Use GCP logging client so python logs can show correct severity level

### PR DESCRIPTION
**Pull request by shifucun**
With this, we should use logging.error()/warning()/info() distinctively to better use the logging filtering.

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
nice, this will be helpful!

File: run_test.sh [hunk 1]
```diff
@@ -186,7 +186,7 @@ function update_integration_test_golden {
 
   export ENV_PREFIX=Autopush
   python3 -m pytest -vv server/integration_tests/topic_cache
-  # python3 -m pytest -vv server/tests/nodejs_e2e_test.py
+  python3 -m pytest -vv server/tests/nodejs_e2e_test.py
```
Comment (pradh) [inline run_test.sh:189]: Thank you for restoring!

File: server/__init__.py [hunk 2]
```diff
@@ -226,6 +225,19 @@ def register_routes_common(app):
 def create_app(nl_root=DEFAULT_NL_ROOT):
   app = Flask(__name__, static_folder='dist', static_url_path='')
 
+  cfg = lib_config.get_config()
+
+  if not cfg.LOCAL:
+    client = google.cloud.logging.Client()
+    client.setup_logging()
+  logging.basicConfig(
```
Comment (pradh) [inline server/__init__.py:233]: future PR is fine, should we do this logging config in a util and reuse from everywhere

**Review by pradh — APPROVED** [review]
Very nice!

File: server/__init__.py (hunk 2 above)
Comment (shifucun) [reply server/__init__.py:233]: The cfg is for the web server only. Now nl does not have it. Should look into a general way to check if running in a GCP environment..

**Pull request merged**